packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +292,49 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   * Add subcommand and sub-sub commands for 'setup' subcommand.
+   * @param {import('argparse').SubParser} subParser
+   */
+  static _addSetupToParser(subParser) {
+    const setupParser = subParser.add_parser('setup', {
+      add_help: true,
+      help: `Install latest official drivers and/or plugins which, ` +
```

Comment:
the comma after `which` is not needed

could work on current host platform ...  for specific usage -> compatible with the current host platform ...